### PR TITLE
Feature/last squid standing

### PR DESCRIPTION
**Pull request by greimers**
Update readme to announce last update.
Update homepage to explain that Tincta was pulled from the App Store for compatibility issues.

### CODE REVIEW

File: Source/Tincta/WhatsNew.txt [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
-
-🎉 10 YEARS OF TINCTA!
+🐙 TINCTA SAYS GOODBYE!
 ======================
-Tincta 1.0 has been released exactly ten years ago on July 09, 2011.
-
-What could be better to celebrate this event than to fix a little bug that has been there ever since version 1.0:
-Finally fixed: Autocompletion of brackets and quotes could only be turned off, by turning off both.
-
+It is over 11 years that the first version of Tincta was released. For an octpus this is an exceptional age (most species live only about one year!).
```
Comment (annaneo) [inline Source/Tincta/WhatsNew.txt:3]: over 11 years ago (oder: It is more then 11 years)

File: Source/Tincta/WhatsNew.txt [hunk 2]
```diff
@@ -1,42 +1,34 @@
...
+However, Tincta is showing signs of ageing. Already on macOS Monterey, we noticed an increase of app crashes. In future versions of macOS, Tincta won’t launch anymore at all. We could neither locate nor fix these crashes.
+Tincta’s codebase is over 10 years old and was originally built for MacOS Snow Leopard. Since them, macOS has moved on not only from cats to landmarks but also technically. But Tincta, being a free app, could not keep up.
```
Comment (annaneo) [inline Source/Tincta/WhatsNew.txt:5]: Since then

File: Source/Tincta/WhatsNew.txt [hunk 3]
```diff
@@ -1,42 +1,34 @@
...
-However, that was also the time when GitHub and Microsoft both brought their free editors Visual Studio Code and Atom to the Mac. Without a doubt, they were more powerful than Tincta and Tincta Pro and we did not stand a chance to compete in features. Luckily, these large IDE editors were also quite slow, heavy and complicated. So we decided to focus on Tincta's simplicity and speed.
-We integrated Tincta Pro's best features back into Tincta and made Tincta free and open source.
-We are sure Tincta still has its place in people's toolbox because it is so approachable and lightweight. You don't always have the patience to wait 10 seconds for some overblown editor to launch. We often rely on Tincta ourselves to jot things down or do quick edits.
+So, it is time to say goodbye to the Kraken. This will be the last update for the time being.
+Maybe some day Tincta will return from the depths if we find time to re-write the code base (or if someone else figures out how to fix those crashes and makes pull request).
```
Comment (jpeinelt) [inline Source/Tincta/WhatsNew.txt:8]: typo: "makes a pull request", but better: "opens a pull request"

**Review by jpeinelt — APPROVED** [review]
just small remarks

**Comment by greimers** [comment]
This branch should only be merged when we actually remove Tincta from the App Store and when we want to update the website.

File: Source/Tincta/WhatsNew.txt [hunk 4]
```diff
@@ -1,42 +1,34 @@
...
-
-🎉 10 YEARS OF TINCTA!
+🐙 TINCTA SAYS GOODBYE!
 ======================
-Tincta 1.0 has been released exactly ten years ago on July 09, 2011.
-
-What could be better to celebrate this event than to fix a little bug that has been there ever since version 1.0:
-Finally fixed: Autocompletion of brackets and quotes could only be turned off, by turning off both.
-
+It has been over 11 years that the first version of Tincta was released. For an octpus this is an exceptional age (most species live only about one year!).
```
Comment (jpeinelt) [inline Source/Tincta/WhatsNew.txt:3]:
```suggestion
It has been over 11 years that the first version of Tincta was released. For an octopus this is an exceptional age (most species live only about one year!).
```

**Review by jpeinelt — APPROVED** [review]
typo

**Pull request merged**